.whitesource

```diff
@@ -0,0 +1,8 @@
+##########################################################
+#### WhiteSource "Bolt for Github" configuration file ####
+##########################################################
+
+# Configuration #
+#---------------#
+ws.repo.scan=true
+vulnerable.check.run.conclusion.level=success
```

Comment:
Leave this file out of the PR since it's not related to the changes to the find module.

Reply:
@mattclay Sorry, I have just fixed it